packages/assets/profiles/htc/htc-vive.json

```diff
@@ -0,0 +1,85 @@
+{
+    "profileId" : "htc-vive",
+    "assets" : {
+        "none" : {
+            "path" : "none.glb",
+            "rootNodeName" : "none-controller-node"
+        },
+        "left" : {
+            "path" : "left.glb",
+            "rootNodeName" : "left-controller-node"
+        },
+        "right" : {
+            "path" : "right.glb",
+            "rootNodeName" : "right-controller-node"
+        }
+    },
+    "layouts": {
+        "left-right-none": {
+            "components" : {
+                "xr-standard-trigger" : {
+                    "rootNodeName" : "SELECT",
+                    "labelAnchorNodeName" : "trigger-label",
+                    "analogButtonValues": true,
+                    "visualResponses": [
+                        {
+                            "rootNodeName" : "SELECT",
+                            "source" : "button",
+                            "states" : ["default", "touched", "pressed"]
+                        }
+                    ]
+                },
+                "xr-standard-grip": {
+                    "rootNodeName" : "GRASP",
+                    "labelAnchorNodeName" : "grip-label",
+                    "visualResponses": [
+                        {
+                            "rootNodeName" : "GRASP",
+                            "source" : "state",
+                            "states" : ["pressed"]
+                        }
+                    ]
+                },
+                "xr-standard-touchpad": {
+                    "rootNodeName" : "TOUCHPAD_PRESS",
+                    "labelAnchorNodeName" : "grip-label",
+                    "touchPointNodeName" : "TOUCH",
+                    "visualResponses": [
+                        {
+                            "rootNodeName": "TOUCHPAD_PRESS",
+                            "source" : "state",
+                            "states" : ["pressed"]
+                        },
+                        {
+                            "rootNodeName": "TOUCH",
+                            "source" : "state",
+                            "states" : ["touched", "pressed"],
+                            "property": "visibility"
+                        },
+                        {
+                            "rootNodeName": "TOUCHPAD_TOUCH_X",
+                            "source" : "xAxis",
+                            "states" : ["default", "touched", "pressed"]
+                        },
+                        {
+                            "rootNodeName": "TOUCHPAD_TOUCH_Y",
+                            "source" : "yAxis",
+                            "states" : ["default", "touched", "pressed"]
+                        }
+                    ]
+                },
+                "menu": {
```

Comment:
According to the WebXR Gamepads module [spec](https://immersive-web.github.io/webxr-gamepads-module/#xr-standard-gamepad-mapping), "Buttons reserved by the UA or platform MUST NOT be exposed." So, I don't think we should have this entry for a menu button unless it has something to do with legacy WebVR compatibility.

Reply:
Agreed for system buttons like Home.  However, Menu is generally available for native apps to use.

Are we agreed on which button a UA may use within each profile for its own UI?  If not, UAs could diverge and leave few buttons consistently available for page use.

Picking the single “UA menu” button for each profile seems like a critical thing for us all to align on in this registry.

Reply:
Since this doesn't appear to have agreement yet, I'll file an issue to discuss whether "reserved buttons" includes buttons the UA will be reserving and how to ensure that is consistent.

Reply:
https://github.com/immersive-web/webxr-input-profiles/issues/82